apps/backend/src/routes/signup/onboard/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<!-- JavaScript code for Continue button -->
+<script lang="ts">
+	let confirm: boolean = false;
+</script>
+
+<!-- Onboard Form - Logo -->
+<div class="skeleton w-32 h-32"></div>
+
+<div class="flex flex-row items-center justify-center h-screen">
+	<!-- Onboard Form - Card -->
+	<div class="w-full max-w-2xl p-10 border border-primary rounded-md shadow-lg">
+		<!-- Onboard Form - Header -->
+		<h1 class="text-3xl font-semibold text-center mb-5">Get Onboard</h1>
+
+		<h2 class="text-center">Complete your profile to start using Armadillo!</h2>
+
+		<form action="">
+			<!-- Onboard Form - Name Input -->
+			<div>
+				<label class="form-control w-full max-w-xl">
+					<div class="label">
+						<span class="label-text">Full Name</span>
+					</div>
+					<input
+						type="text"
+						name="fullname"
+						placeholder="John Doe"
+						class="input input-bordered w-full max-w-xl
+					focus:ring-primary focus:border-primary duration-300 focus:outline-none"
+					/>
+				</label>
+			</div>
+
+			<!-- Onboard Form - Corporate Domain Address Input -->
+			<div>
+				<label class="form-control w-full max-w-xl">
+					<div class="label">
+						<span class="label-text">Domain Address</span>
+					</div>
+					<input
+						type="text"
+						name="domainaddress"
+						placeholder="nyp.edu.sg"
+						class="input input-bordered w-full max-w-xl
+					focus:ring-primary focus:border-primary duration-300 focus:outline-none"
+					/>
+				</label>
+			</div>
+
+			<!-- Onboard Form - Headshot Photo Input -->
+			<div>
+				<label class="form-control w-full max-w-xl">
+					<div class="label">
+						<span class="label-text">Headshot Photo (.png/.jpg)</span>
+					</div>
+					<input
+						type="file"
+						name="headshot"
+						class="file-input file-input-bordered file-input-primary w-full max-w-xl"
+					/>
```

Comment:
Missing `accept` attribute

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

Reply:
added accept="image/png, image/jpeg" for now but, this will be edited in the future as the headshot photo will be taken separate from the onboarding process so for now, ill leave it as it is.